sycl/test-e2e/ESIMD/lit.local.cfg

```diff
@@ -5,3 +5,6 @@ config.required_features += ['gpu']
 
 if 'gpu-intel-gen9' in config.available_features and platform.system() == 'Windows':
   config.unsupported = True
+
+if 'gpu-intel-dg2' in config.available_features:
+  config.required_features += ['level_zero']
```

Comment:
We need this to fix failures when run on OCL. The current postcommit job only runs L0 anyway, so there's no difference in coverage. We should investigate why OCL fails separately.

Reply:
That comment should go into the source file :)